Makefile

```diff
@@ -21,13 +21,15 @@ compiler: bin/act
 
 test_specs=$(wildcard tests/*/*.act)
 
-failing_typing=tests/array/array.act tests/dss/vat.act tests/creation/createMultiple.act
-
-typing_specs=$(filter-out $(failing_typing), $(test_specs))
+parser_specs=$(filter-out $(invariant_specs), $(test_specs))
```

Comment:
why not test the parsing of invariants?

Reply:
hmmmm, I just didn't want to clutter up the tests folder with a bunch of parsed output...

Reply:
I'm not sure how valuable I find the parsing tests atm anyway, the output is extememly hard to diff and read. I would be more in favour of removing them and fuzzing the parser instead...

Reply:
Ok, that's alright with me